server/plugin/api.go

```diff
@@ -1285,17 +1286,17 @@ func (p *Plugin) getReposByOrg(c *UserContext, w http.ResponseWriter, r *http.Re
 	org := r.URL.Query().Get("organization")
 
 	if org == "" {
-		err := errors.New("organization query param is empty")
-		c.Log.WithError(err).Warnf("Failed to list repositories")
-		p.writeAPIError(w, &APIErrorResponse{Message: "Failed to fetch repositories", StatusCode: http.StatusInternalServerError})
+		err := errors.New("Organization query param is empty")
+		c.Log.WithError(err).Warnf("Organization query param is empty")
```

Comment:
Why have we made this change? We will be printing the same message twice in the logs with this change.

Reply:
You're right, I've changed it to the following: 

```suggestion
		c.Log.Warnf("Organization query param is empty")
```

 I'm not sure what best practices are here but I haven't seen an example of context logging without an error. Would this be okay?